app/services/calculators/subject_matter_of_dispute_disregard_calculator.rb

```diff
@@ -1,30 +1,28 @@
+# WARNING: This calculator assumes that the assessed value/equity of all disputed properties and vehicles
+# has already been calculated. If this is not the case, it will produce inaccurate results.
 module Calculators
```

Comment:
I think we're (still) missing a test of the top level that proves that everything is joined up properly. This comment just shows how fragile the current structure is - effectively you can't tell at level 2 whether level 1 has done the calculations you require

Reply:
Good point!